src/main/java/org/onebusaway/alexa/AnonSpeechlet.java

```diff
@@ -261,4 +266,23 @@ private SpeechletResponse askForCity(Optional<String> currentCityName) {
         }
         return SpeechletResponse.newAskResponse(out, cityReprompt);
     }
+
+    private String allRegionsSpoken() {
+        List<String> activeRegions = obaClient.getAllRegions()
+                .stream()
+                .filter(r -> RegionUtils.isRegionUsable(r) && !r.getExperimental())
```

Comment:
To be consistent with the other code we should also check if `r.getObaBaseUrl() == null`, and if so remove that region as well.